upup/pkg/kutil/delete_cluster.go

```diff
@@ -1213,6 +1215,14 @@ func DeleteVPC(cloud fi.Cloud, r *ResourceTracker) error {
 	return nil
 }
 
+func DumpVPC(r *ResourceTracker) (interface{}, error) {
```

Comment:
Looks like we are using the `delete` cluster for something here.. which is awesome..

Right now kops is able to find cluster resources to delete by going through and querying AWS for tags, and resource names.. So it seems like a lot of the cluster dump overhead might already be here.. my only thought on the matter is that the cluster definition (dump) should come **before** the delete.

In other words..

We have the cluster dump (a way to poll AWS for resources) that can be used in many ways, and delete is simply a wrapper that uses the dump, as well as outputting the dump to json, yaml, etc..

Just a thought 😄